examples/src/components/utils/Menu.jsx

```diff
@@ -0,0 +1,112 @@
+// (C) 2007-2018 GoodData Corporation
+/* eslint-disable react/jsx-closing-tag-location */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { NavLink, withRouter } from 'react-router-dom';
+
+class Menu extends React.Component {
+    render() {
+        const { location: { pathname }, sideNavigationRoutes, routes, isUserLoggedIn } = this.props;
+        const href = pathname;
+        const currentRoute = (href !== undefined && routes.find(link => (link.path === BASEPATH + href))) || null;
+
+        if (!isUserLoggedIn) {
```

Comment:
Do we really want to forbid users to go to the documentation for example if they are not logged in?